### PR TITLE
Add alt text to images on studio homepage

### PR DESCRIPTION
**Pull request by megcrenshaw**
Adds alt text on studio teacher homepage, [studio.code.org/home](https://studio.code.org/home). There's some interesting discussion here https://codedotorg.slack.com/archives/C9MB4CL07/p1666194895955709?thread_ts=1666188684.238069&cid=C9MB4CL07 asking if we should even have alt text for these, esp. for the image on the bottom left (for CS Journeys). Are these just decoration? Do we have 3 of the 4 images with alt text? Curious to hear what y'all think!

There should be no changes visually:
Screen Shot 2022-10-19 at 2 36 17 PM

I checked the alt text appears for each image, e.g.:
Screen Shot 2022-10-19 at 2 36 57 PM

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/i18n/common/en_us.json [hunk 1]
```diff
@@ -527,6 +527,7 @@
   "csfExpressTitle": "CS Fundamentals Express",
   "csJourneys": "CS Journeys",
   "csJourneysDescription": "Bring CS to life and help students make real world connections to what they're learning. Empower them to continue their CS journeys with these resources.",
+  "csJourneysImageDescription": "Icons of a star, a rocket, and a thumbs up sign with text 'CS Journeys'",
```
Comment (hannahbergam) [inline apps/i18n/common/en_us.json:530]: I think my vote would be to leave this one empty

File: apps/i18n/common/en_us.json [hunk 2]
```diff
@@ -1658,6 +1659,7 @@
   "print": "Print",
   "privacyPolicy": "Privacy Policy",
   "privacyPracticesForChildren": "See our privacy practices for children",
+  "professionalLearningImageDescription": "A focused facilitator wearing a Code.org name tag is working at a table during a workshop.",
```
Comment (hannahbergam) [inline apps/i18n/common/en_us.json:1662]: Love the 'focused' descriptor here

File: apps/i18n/common/en_us.json [hunk 3]
```diff
@@ -2070,6 +2073,7 @@
   "teacherComments": "Teacher comments",
   "teacherCommunity": "Teacher Community",
   "teacherCommunityDescription": "Connect with other teachers on the forum to ask curriculum questions, share ideas from your lessons, and get help from the Code.org community.",
+  "teacherCommunityImageDescription": "Two teachers smile and pose for the camera, touching two apples together as if they were clinking glasses for a toast.",
```
Comment (hannahbergam) [inline apps/i18n/common/en_us.json:2076]: *chefs kiss*

**Review by hannahbergam — APPROVED** [review]
Thanks for doing this! I'm happy with this as-is, but the icon photo might be skip-over-able.

**Review by davidsbailey — APPROVED** [review]
❤️

File: apps/i18n/common/en_us.json (hunk 2 above)
Comment (cat5inthecradle) [reply apps/i18n/common/en_us.json:1662]: We use active voice elsewhere: "works" instead of "is working"

**Review by cat5inthecradle — COMMENTED** [review]
Is there a way to support all three possibilities for alt text?

* "a string of text"
* "" (an empty string)
* (no alt text)

**Comment by megcrenshaw** [comment]
@cat5inthecradle great question –– I think very soon, I'm going to make a video where we can hear how they each sound on a screen reader. Stay tuned! With the caveat that each screen reader is slightly different ...

**Comment by megcrenshaw** [comment]
@epeach  Curious to hear your thoughts on if any and/or all of these images just have alt text of `""` instead

**Comment by megcrenshaw** [comment]
Clip of how the screenreader reads the different alt texts:

https://user-images.githubusercontent.com/9142121/197040449-7701cc96-2afc-4c9e-becf-073c2fc44d8d.mov

The big takeaway is that leaving the alt text field completely blank on an image is pretty brutal, but adding a "" is reasonable.

**Comment by megcrenshaw** [comment]
After a chat with Alice, Hannah, and Mario (plus input from Darin), I'm closing this PR in favor of letting the default prop add an empty string for alt text since these images seem more decorative than informative.

**Pull request closed without merging**